src/web/app/user-join-page.component.spec.ts

```diff
@@ -308,4 +317,42 @@ describe('UserJoinPageComponent creating account', () => {
     expect(component.isLoading).toBeFalsy();
     expect(component.validUrl).toBeFalsy();
   });
+
+  it('should show error message if 5xx is returned when joining course', () => {
```

Comment:
These testcases are still in the wrong place. They should be grouped with the other testcases for join course, not for the testcases for create account.